src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -405,6 +405,19 @@ export function getUiSettingDefaults() {
       }),
       category: ['search'],
     },
+    'courier:batchSearches': {
+      name: i18n.translate('kbn.advancedSettings.courier.batchSearchesTitle', {
+        defaultMessage: 'Batch concurrent searches in a single requests',
+      }),
+      value: false,
+      type: 'boolean',
+      description: i18n.translate('kbn.advancedSettings.courier.batchSearchesText', {
+        defaultMessage:
+          'Batch multiple concurrent searches in a single request. This may improve overall load' +
+          'times, but it also means that no single search request will complete until the last has completed.',
```

Comment:
Or maybe:
```suggestion
          'times, but it also means that no single search request will be returned until the last has completed.',
```

TBH either way makes sense to me... just trying to think of how to explain this in the clearest possible terms.